server/models/schemas.js

```diff
@@ -172,6 +172,22 @@ var payments = new mongoose.Schema({
     attrs: {}
 },{timestamps: true});
 
+var expenseMaster = new mongoose.Schema({
+    expenseName: String,
+    updatedBy: String,
+    createdBy: String,
+    attrs: {}
+},{timestamps: String});
+
+var paymentsReceived = mongoose.Schema({
+    tripId: String,
+    driverId: String,
```

Comment:
I think this should truckId not driverId. Also why are we not using references? Also missing accountId